forem/forem
# Update dev.to/mods for non-logged in users

## What type of PR is this? (check all applicable)


## Description
Update the copy on `dev.to/mod`, which is currently confusing for folks who are not already mods (trusted users or tag mods).

## Related Tickets & Documents
closes https://github.com/forem/forem/issues/11851

## QA Instructions, Screenshots, Recordings
1. Navigate to `localhost:3000/mod` as a user that is not trusted or a tag moderator.
2. You should see copy such as:
```
We periodically award some DEV members with heightened privileges to help moderate the community.
Check out our Code of Conduct and read through our Trusted User Guide and Tag Moderation Guide.
If you'd like to assist us as a trusted user or tag mod, please email us at yo@dev.to and let us know which role you're interested in and why. If it's tag moderation, please tell us what tags you'd like to moderate for.
```
3. Repeat step 1 on a forem that is NOT DEV. The page copy **should reflect the unique name and email of the forem being viewed**.


### UI accessibility concerns?

_No UI changes_

## Added tests?


## Added to documentation?

      [Admin Guide](https://forem.gitbook.io/forem-admin-guide/)

## [optional] Are there any post deployment tasks we need to perform?
None

## [optional] What gif best describes this PR or how it makes you feel?

![Cartoon](https://media.giphy.com/media/3o72F241sDzoHEWCzu/giphy.gif)
